src/TestImages.jl

```diff
@@ -1,25 +0,0 @@
-module TestImages
-using Images
-
-export testimage
-
-const imagedir = joinpath(Pkg.dir(), "TestImages", "images")
-
-function testimage(filename, ops...)
```

Comment:
The ```master``` branch is untouched (Thus the size of the package is still small \m/). All the original source files are still there. So whenever a user adds the ```TestImages``` package, he will get the original source. Once this PR is merged, I will open another PR for the master branch changing the links in the original source files to those in this repository. Now when the user will do ```Pkg.build("TestImages")```, they will be downloaded from this repository. All other functions like ```testimage()``` work like before. The ```gh-pages``` branch will just host the website and the images. It does not need the actual package files.

I feel the ```testimage``` function should also give an option to check the repository if the image is not present locally. If it still cannot find it, then the error should be displayed. I''ll open a PR for the same once this is merged.

Once merged, the site will be live at timholy.github.io/TestImages.jl.